service/controller/v13/resource/deployment/template/kubernetes_load_balancer_setup.json

```diff
@@ -11,13 +11,14 @@
     }
   },
   "variables":{
-    "loadBalancerSkuName": "Standard"
+    "loadBalancerSkuName": "Standard",
+    "loadBalancerPIPName": "[parameters('ingressPIPName')]"
```

Comment:
Can't we directly use the parameter instead of using a variable?

Reply:
Do you mean dropping this and changing the line below to
`        "[concat('Microsoft.Network/publicIPAddresses/', parameters('ingressPIPName'))]"` :question:

Reply:
Yes